<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
background-color: rgba(0, 0, 0, 191);
border-radius:15px;
}
#LoginText{
color:white;
background:transparent;
font-family:century gothic;
font-size:24px;
}
#UsernameText{
color:white;
background:transparent;
font-family:century gothic;
font-size:24px;
}
#PasswordText{
color:white;
background:transparent;
font-family:century gothic;
font-size:24px;
}
#IC_Teller{
color:white;
background:transparent;
font-family:century gothic;
font-size:18px;
}
#Login{
background:url(:/FINAL.png)
}
QToolButton{
background:red;
border-radius:60px;
}
#eye{
background:transparent;
border:none;
}
#eye:hover{
background:red;
}
#LoginButton{
background:red;
color:white;
border-radius:15px;
font-family:century gothic;
font-size:24px;
}
#LoginButton:hover
{
background:#49ebff;
color:#333;
border-radius:15px;
font-family:century gothic;
font-size:24px;
}
QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
font-family:century gothic;
font-size:24px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>421</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="LoginText">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string> LOGIN HERE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EnterUsername">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EnterPassword">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QLabel" name="UsernameText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="IC_Teller">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>449</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="eye">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>403</y>
       <width>41</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pics.qrc">
       <normaloff>:/eyes.png</normaloff>:/eyes.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="pics.qrc">
      <normaloff>:/image.png</normaloff>:/image.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
